_posts/2024-08-16-logdetective-reviews.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Reviewing annotations in Log Detective"
+author: "Tomas Tomecek"
+layout: "post"
+---
+
+We appreciate everyone who submitted annotations, thank you! It will allow us to
+create a model with knowledge about building software, not just RPMs, but also
+container images, test failures and various language specific buildsystems. I
+envision that Log Detective will provide useful answers to most of logs with
+build failures. Though we still have a long way getting there.
```

Comment:
I don't expect all Copr users to immediately know what Log Detective is.

I suggest to edit the first paragraph to:

```
You may have heard of our [LogDetective](https://logdetective.com/) project.
We want to use AI to process your log of failed builds, show you where
the problem is and how to fix it. We have enough failure logs, but we need
to supplement them with notes on why the build went wrong and how to fix it. 
```

Reply:
thanks, will add that!